core/src/main/kotlin/com/malinskiy/marathon/cache/test/key/TestCacheKeyFactory.kt

```diff
@@ -25,7 +22,6 @@ class TestCacheKeyFactory(
             .use {
                 it.write(versionNameProvider.versionName)
```

Comment:
Maybe we can remove `versionNameProvider.versionName` as well? The version change shouldn't trigger test cache wipeout by itself. And if the apk will change then `componentCachingKey` will change as well.

Reply:
The idea might've been for the cache key to depend on the version of Marathon itself, however `versionName ` seems to always be `unknown` because there is no `Implementation-Version:` in the JAR manifest.